app/config/AppConfig.scala

```diff
@@ -71,7 +71,14 @@ class AppConfig @Inject()(val config: Configuration, servicesConfig: ServicesCon
 
   lazy val whatYouOweUrl: String = config.get[String]("urls.whatYouOwe")
 
-  lazy val feedbackUrl: String = config.get[String]("urls.feedbackUrl")
+  lazy val contactFrontendHost: String = s"${servicesConfig.baseUrl("contact-frontend")}/"
+  val contactFrontendStartUrl: String = "feedback.url"
+  val contactFrontendServiceId: String = "contact-frontend.serviceId"
+  lazy val feedbackUrl: String =
+    contactFrontendHost +
+      config.get[String](contactFrontendStartUrl) +
+      "?service=" + config.get[String](contactFrontendServiceId) +
+      s"&backUrl=" + config.get[String]("host") + "/penalties"
```

Comment:
The back Url should be whatever page the user was previously on (not just the penalties homepage) see this PR for an example implementation https://github.com/hmrc/penalties-frontend/pull/240